bot/mozdata.ini

```diff
@@ -0,0 +1,2 @@
+[User-Agent]
+name=mozilla/code-review/bot
```

Comment:
It'd be ideal to follow the same User-Agent we're setting at https://github.com/mozilla/code-review/blob/978802d2dd451462bb4b62977f7d3fe326312d5d/bot/code_review_bot/config.py#L32